backend/src/main/java/com/sim_backend/websockets/GsonUtilities.java

```diff
@@ -23,6 +24,8 @@ public static Gson getGson() {
         GsonBuilder gsonBuilder = new GsonBuilder();
```

Comment:
Something I noticed in the GsonBuilder documentation:
`By default, Gson omits all fields that are null during serialization.`

The above quote may not be relevant (I don't think we're initializing fields as null ever?). However, for messages with no payload, are we serializing them with {} or null? The ocpp-j 1.6 spec requires one of them:
```Payload is a JSON object containing the results of the executed Action. If there is no payload JSON allows for two different notations: null or and empty object \{}. Although it seems trivial we consider it good practice to only use the empty object statement. Null usually represents something undefined, which is not the same as empty, and also \{} is shorter.```